src/mudlet-lua/lua/GUIUtils.lua

```diff
@@ -2073,20 +2078,56 @@ function creplace(window, text)
   xReplace(window, text, 'c')
 end
 
+--- version of replaceLine function that allows for color, by way of cinsertText
+--- @param windowName Optional name of the window to replace on
+--- @param text The text to replace the selection with.
+function creplaceLine(window, text)
+	if not text then
```

Comment:
There's an indentation error here.

Reply:
Argh. copypasta error from the original creplaceLine. It all showed up the same in VSCode. Thanks for catching it.